doc/manual/source/threat-model.rst

```diff
@@ -0,0 +1,72 @@
+Threat Model
+============
+
+Based on the `threat model by restic <https://github.com/restic/restic/blob/master/doc/design.rst#threat-model>`_, we present the threat model for Routinator.
+
+What is Routinator?
+-------------------
+
+Routinator is an RPKI (Resource Public Key Infrastructure) relying party client that retrieves data from the various RPKI repositories, checks its validity, and produces a validated payload for consumption by BGP (Border Gateway Protocol) routers. 
+
+BGP routers are the backbone of the Internet. They provide the protocols that allow your packet to reach its destination. This is done by networks announcing to neighbouring networks what destination can be reached via them. Traditionally this was done on good faith -- the RPKI adds a way to partially verify these statements.
+
+The data Routinator retrieves from the repositories, as well as the repositories themselves, are untrusted. Objects are validated using cryptographic signatures that can only be made by resource holders, and contents are checked according to the schema, and data is checked against the constraints in the certificate.
```

Comment:
You say the data is untrusted then seem to go on to describe it as validated and checked, which to me at least seems like a contradiction. Perhaps you should say "Trust is established by ...".